Figure_02_01.py

```diff
@@ -1,16 +1,16 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
-w = 0.2
+w = 0.2 #rad/s
 s = w*1j
 G = 5*np.exp(-2*s)/(10*s + 1)
 
 t = np.linspace(0, 100, 200)
-GM = np.abs(G)
-PM = np.angle(G)
+GN = np.abs(G)      #Gain at w = 0.2 rad/s
```

Comment:
OK, I'm happy with the reasoning about the name change. I don't like embedding the value of a variable in a comment, though, also 

- [ ] Change comment to follow PEP8